company-coq.el

```diff
@@ -4262,10 +4262,12 @@ REF-BUFFER is used to retrieve the buffer-local values of
 
 (defun company-coq-features/prettify-symbols--same-ish-syntax (other ref)
   "Check if the syntax class of OTHER is similar to that of REF."
-  (memq (char-syntax (or other ?\s))
-        (if (memq (char-syntax ref) '(?w ?_))
-            '(?w ?_)
-          '(?. ?\\))))
+  (and
+   (not (memq other '(44 46 59 124))) ;; , . ; |
```

Comment:
You can use `?.`, `?,`, `?\;`, and `?|` for better readability.
But I'm not sure about teh last one (`|`); won't it cause `|||` to be prettified as `||` followed by `|`?

Reply:
I removed ?|. Sadly [->|] doesn't work now.